Can't use dotnet v8.0.411 to build sample 2 (static text)

When trying to build sample 2 (https://github.com/drewnoakes/figgle/tree/master/samples/2-static-text) with dotnet v8.0.411, it fails:
```
C:\Users\Jeroen\Downloads\FiggleTest>dotnet --version
8.0.411

C:\Users\Jeroen\Downloads\FiggleTest>dotnet build
  Determining projects to restore...
  All projects are up-to-date for restore.
CSC : warning CS9057: The analyzer assembly 'C:\Users\Jeroen\.nuget\packages\figgle.generator\0.6.4\analyzers\dotnet\cs\Figgle.Generator.dll' references version '4.14.0.0' of the compiler, which is newer than the currently running version '4.11.0.0'. [C:\Users\Jeroen\Downloads\FiggleTest\2-static-text\Figgle.Samples.StaticText.csproj]
C:\Users\Jeroen\Downloads\FiggleTest\2-static-text\Program.cs(1,7): error CS0246: The type or namespace name 'Figgle' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\Jeroen\Downloads\FiggleTest\2-static-text\Figgle.Samples.StaticText.csproj]
C:\Users\Jeroen\Downloads\FiggleTest\2-static-text\Program.cs(5,2): error CS0246: The type or namespace name 'GenerateFiggleTextAttribute' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\Jeroen\Downloads\FiggleTest\2-static-text\Figgle.Samples.StaticText.csproj]
C:\Users\Jeroen\Downloads\FiggleTest\2-static-text\Program.cs(5,2): error CS0246: The type or namespace name 'GenerateFiggleText' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\Jeroen\Downloads\FiggleTest\2-static-text\Figgle.Samples.StaticText.csproj]

Build FAILED.

CSC : warning CS9057: The analyzer assembly 'C:\Users\Jeroen\.nuget\packages\figgle.generator\0.6.4\analyzers\dotnet\cs\Figgle.Generator.dll' references version '4.14.0.0' of the compiler, which is newer than the currently running version '4.11.0.0'. [C:\Users\Jeroen\Downloads\FiggleTest\2-static-text\Figgle.Samples.StaticText.csproj]
C:\Users\Jeroen\Downloads\FiggleTest\2-static-text\Program.cs(1,7): error CS0246: The type or namespace name 'Figgle' could not be found (are ou missing a using directive or an assembly reference?) [C:\Users\Jeroen\Downloads\FiggleTest\2-static-text\Figgle.Samples.StaticText.csproj]
C:\Users\Jeroen\Downloads\FiggleTest\2-static-text\Program.cs(5,2): error CS0246: The type or namespace name 'GenerateFiggleTextAttribute' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\Jeroen\Downloads\FiggleTest\2-static-text\Figgle.Samples.StaticText.csproj]
C:\Users\Jeroen\Downloads\FiggleTest\2-static-text\Program.cs(5,2): error CS0246: The type or namespace name 'GenerateFiggleText' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\Jeroen\Downloads\FiggleTest\2-static-text\Figgle.Samples.StaticText.csproj]
    1 Warning(s)
    3 Error(s)

Time Elapsed 00:00:01.67
```

With dotnet v9.0.301, it succeeds:
```
G:\FiggleTest>dotnet --version
9.0.301

G:\FiggleTest>dotnet build
Restore complete (0,3s)
  Figgle.Samples.StaticText succeeded (0,1s) → 2-static-text\bin\Debug\net8.0\Figgle.Samples.StaticText.dll

Build succeeded in 0,6s
```

--- a/Directory.Packages.props
+++ b/Directory.Packages.props
@@ -7,8 +7,8 @@
 
   <ItemGroup>
     <PackageVersion Include="IsExternalInit" Version="1.0.3" />
-    <PackageVersion Include="Microsoft.CodeAnalysis.Analyzers" Version="4.14.0" />
-    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="4.14.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.3" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="4.3.1" />
     <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="8.0.0" />
     <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
     <PackageVersion Include="Roslyn.Diagnostics.Analyzers" Version="4.14.0" />

--- a/src/Figgle.Fonts.Generator/FiggleFontGenerator.cs
+++ b/src/Figgle.Fonts.Generator/FiggleFontGenerator.cs
@@ -50,7 +50,7 @@ internal sealed class FiggleFontGenerator : IIncrementalGenerator
                 var csvFileContent = file.GetText(cancellationToken)?.ToString();
                 if (csvFileContent is null)
                 {
-                    return [];
+                    return ImmutableArray<FontAlias>.Empty;
                 }
 
                 var aliases = ImmutableArray.CreateBuilder<FontAlias>();

